### PR TITLE
Safer handling of getting horizontal content item fetch more results

### PR DESCRIPTION
**Pull request by conrad-vanl**
### What does this PR do, or why is it needed?

Have been getting off-and-on errors navigating content and content children on The Porch App. Such as:

Screen Shot 2020-03-16 at 2 40 24 PM

and 

Screen Shot 2020-03-16 at 2 41 06 PM

Looking at network stacks, it appears that all queries were successful. I am unsure why `null` is being returned in this scenario yet, but it seems like something the UI should handle without throwing errors. Thoughts?

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by conrad-vanl** [comment]
Needs someone to run `--fixlint`. Next time I have this repo pulled up I'll run it...otherwise whoever reviews this if you could be so kind 😅😇

File: packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ class HorizontalContentSeriesFeedConnected extends Component {
               const connection = isParent
                 ? 'childContentItemsConnection'
                 : 'siblingContentItemsConnection';
-              const newEdges = get(fetchMoreResult.node, connection, []).edges;
+              const newEdges = get(fetchMoreResult, `node.${connection}.edges`) || []);
```
Comment (vinnyjth) [inline packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: @conrad-vanl any reason you didn't continue to rely on the `default` arg of `get? 
aka
```suggestion
              const newEdges = get(fetchMoreResult, `node.${connection}.edges`, []));
```

Comment (conrad-vanl) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: because it doesn't work the same way. The `default` arg of `get` is only returned if the value is `undefined`, wheres ` || []` returns `[]` if the value is `falsey`.

If we depend on an array later on in the code, you should _not_ use the default arg of `get`.

Comment (vinnyjth) [reply packages/apollos-ui-connected/src/HorizontalContentSeriesFeedConnected/HorizontalContentSeriesFeedConnected.js]: 👍 Makes perfect sense. That nuance of the default value has bit me before.
